android/src/main/kotlin/live/hms/hmssdk_flutter/methods/HMSPipAction.kt

```diff
@@ -99,5 +102,15 @@ class HMSPipAction {
                 activity.enterPictureInPictureMode(params.build())
             }
         }
+
+        /**
+         * This method only needs to be called when application is using PIP mode
+         * [isPIPEnabled] variable keeps a track whether PIP was enabled in application or not
+         */
+        fun disposePIP(activity: Activity) {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S && isPIPEnabled) {
+                activity.setPictureInPictureParams(PictureInPictureParams.Builder().setAutoEnterEnabled(false).build())
```

Comment:
`isPIPEnabled` should be reset to false here?
```suggestion
                activity.setPictureInPictureParams(PictureInPictureParams.Builder().setAutoEnterEnabled(false).build())
isPIPEnabled = false
```